README.md

```diff
@@ -146,4 +151,6 @@ Send in PRs to this repository and feel free to add your name to the authors lis
 
 ## Credits for images:
 
-[Trekmarvel](https://github.com/Trekmarvel) for the base potion bottle graphics.
+* [Trekmarvel](https://github.com/Trekmarvel) for the base potion bottle graphics.
+* InvinciblePotionBottle.png, Regen2PotionBottle.png edited from base by Minege
+* MegaJumpBottle.png, GigaJumpBottle.png edited from base by Patrick Wang
```

Comment:
After this line you could put a quick note that "BlindWalkPotionBottle.png edited from base by digitalripperynr"